models/stg_snapchat_ads__campaign_hourly_report.sql

```diff
@@ -0,0 +1,50 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_snapchat_ads__campaign_hourly_report_tmp') }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_snapchat_ads__campaign_hourly_report_tmp')),
+                staging_columns=get_campaign_hourly_report_columns()
+            )
+        }}
+    from base
+),
+
+final as (
+    
+    select 
+        campaign_id,
+        cast (date as {{ dbt_utils.type_timestamp() }}) as date_hour,
+        attachment_quartile_1,
+        attachment_quartile_2,
+        attachment_quartile_3,
+        (attachment_total_view_time_millis / 1000000.0) as attachment_total_view_time,
+        attachment_view_completion,
+        quartile_1,
+        quartile_2,
+        quartile_3,
+        saves,
+        shares,
+        (screen_time_millis / 1000000.0) as screen_time,
+        video_views,
+        view_completion,
+        (view_time_millis / 1000000.0) as view_time,
+        impressions,
+        (spend / 1000000.0) as spend,
+        swipes
+        
+        {% for metric in var('snapchat_ads__campaign_hourly_report_passthrough_metrics', []) %}
+        , {{ metric }}
+        {% endfor %}
```

Comment:
You will want to add this same logic to the `fields` CTE. Can you do this for each of the models that allow for passthrough metrics. [Here](https://fivetran.slack.com/archives/C02FG7499UK/p1658423688619259) is the example.

If the metric @fivetran-jamie is working on is successful then we will be able to adjust this. But for now we will want to add this into the package.